betel/categories_enum.py

```diff
@@ -0,0 +1,59 @@
+import enum
```

Comment:
just name this play_store_cateories.py (no enum please)

Reply:
Done